vdbench_wrapper/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM quay.io/avili/vdbench:1.0.1
+
+USER root
```

Comment:
don't think this line is needed, I don't use it.

Reply:
I delete it.